libs/wingsdk/test/target-sim/queue.test.ts

```diff
@@ -200,7 +199,7 @@ test("messages are not requeued if the function fails before timeout", async ()
   const app = new SimApp();
   const handler = Testing.makeHandler(INFLIGHT_CODE);
   const queue = new cloud.Queue(app, "my_queue", {
-    timeout: Duration.fromSeconds(30),
+    // timeout: Duration.fromSeconds(30),
```

Comment:
The timeout seems important for the test, should this test be skipped too? If so this can be uncommented